src/Bmotion/Bit.Bmotion.Demo/Server/wwwroot/css/app.css

```
@@ -1,53 +1,1077 @@
+    width: 2rem;
+    align-items: center;
+    justify-content: center;
+    border-radius: 50%;
+    color: var(--bit-clr-pri-text);
+    background-color: var(--bit-clr-pri);
+}
+
+.toggle-theme-btn svg {
+    width: 1.0625rem;
+    height: 1.0625rem;
+    fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)